agents/agent_context.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, settings: AgentSettings, agent_id = None, world_context:List[
         self.settings = settings
         self.agent_id = agent_id
         self.envs = envs
+        self.include_world_processing = include_world_processing
```

Comment:
New arg for world context processing (sometimes unnecessary in more direct tasks.)